frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -210,7 +210,7 @@ const getOperandStatus = (obj: K8sResourceKind): OperandStatusType => {
     };
   }
 
-  const trueCondition = conditions?.find((c: K8sResourceCondition) => c.status === 'True');
+  const trueCondition = _.findLast(conditions, (c: K8sResourceCondition) => c.status === 'True');
```

Comment:
We probably want to sort by update or transition time. The order in the array shouldn't really matter. In fact, the items that are listed first might be more important than those listed last. It depends on the operator.

@tlwu2013

Reply:
> We probably want to sort by update or transition time.

I realize it wouldn't change the behavior for this specific operator since the timestamps match :/ But it might be better for the general case.

Reply:
On the operand details page, we simply display the conditions as they appear in the array.  See https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx#L473 and https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx#L588.  So we need to apply sorting there, too, no?

Reply:
And if we do apply sorting, won't that differ from the CLI?  
``` oc describe AppsodyApplication example-appsodyapplication -n openshift-operators
Name:         example-appsodyapplication
Namespace:    openshift-operators
Labels:       <none>
Annotations:  <none>
API Version:  appsody.dev/v1beta1
Kind:         AppsodyApplication
Metadata:
  Creation Timestamp:  2020-05-06T14:32:16Z
  Generation:          2
  Resource Version:    71810
  Self Link:           /apis/appsody.dev/v1beta1/namespaces/openshift-operators/appsodyapplications/example-appsodyapplication
  UID:                 3684debc-2b33-40b2-b321-160eff7e12c3
Spec:
  Application Image:  registry.connect.redhat.com/ibm/open-liberty-samples:springPetClinic
  Application Name:   example-appsodyapplication
  Pull Policy:        IfNotPresent
  Resource Constraints:
  Service:
    Port:  3000
    Type:  ClusterIP
  Stack:   java-openliberty
Status:
  Conditions:
    Last Transition Time:  2020-05-06T14:32:16Z
    Last Update Time:      2020-05-06T14:32:16Z
    Status:                True
    Type:                  DependenciesSatisfied
    Last Transition Time:  2020-05-06T14:32:17Z
    Last Update Time:      2020-05-06T14:32:17Z
    Status:                True
    Type:                  Reconciled
  Image Reference:         registry.connect.redhat.com/ibm/open-liberty-samples:springPetClinic
Events:                    <none>```

Reply:
I'd argue against sorting the table by transition time since the rows could swap places on updates, which might be weird. I think it's really more of an issue here where we're trying to boil down the status to one thing, and we don't have a good way to know which condition to show.

I wonder if we shouldn't show all the `True` conditions as the status. For instance,

```
Conditions: DependenciesSatisfied, Reconciled
```